### PR TITLE
Use 64bit adopted constant

### PR DESCRIPTION
**Pull request by cosmo0920**
Based on
https://github.com/cosmo0920/windows-pr/issues/13#issuecomment-226088746
and
https://github.com/cosmo0920/windows-pr/issues/13#issuecomment-227010750
patches.

### CODE REVIEW

File: lib/windows/handle.rb [hunk 1]
```diff
@@ -9,7 +9,7 @@ module Handle
 
     private
 
-    INVALID_HANDLE_VALUE           = 0xFFFFFFFF
+    INVALID_HANDLE_VALUE           = (1<<['a'].pack('p').length*8)-1
```
Comment (djberg96) [inline lib/windows/handle.rb]: I don't think that actually works. I always get 0xffffffff, regardless of whether i'm using 32-bit or 64-bit Ruby. Have you tested?

Comment (phasis68) [inline lib/windows/handle.rb]: I confirmed it works.

C:\Ruby22-x64\bin>irb
irb(main):001:0>  (1<<['a'].pack('p').length*8)-1
=> 18446744073709551615

C:\Ruby22\bin>irb
irb(main):001:0>  (1<<['a'].pack('p').length*8)-1
=> 4294967295

Comment (cosmo0920) [inline lib/windows/handle.rb]: > regardless of whether i'm using 32-bit or 64-bit Ruby

Really? I got 0xFFFFFFFFFFFFFFFF when I use Ruby 2.3 x64
AppVeyor CI passes for i386 and x64 Rubies even if using this code: https://ci.appveyor.com/project/cosmo0920/windows-pr/build/4

**Comment by cosmo0920** [comment]
But current path does not support JRuby.
My suggestion is as follows:

In 1.2.x series, it should not include big changes.
So, I don't want to support JRuby for 1.2.x patch release.
For 1.3.0 or later, using ffi code is acceptable. (TODO for future plans)

File: lib/windows/handle.rb (hunk 1 above)
Comment (djberg96) [inline lib/windows/handle.rb]: Hm, maybe I had my environment messed up.

**Comment by cosmo0920** [comment]
Can I merge this?
I want to release new version of windows-pr which includes this patch.

**Comment by djberg96** [comment]
It's your project, you can merge it if you want. :)

:+1:

**Pull request merged**